yazi-boot/src/args.rs

```diff
@@ -6,8 +6,8 @@ use clap::{command, Parser};
 #[command(name = "yazi")]
 pub struct Args {
 	/// Set the current working entry
-	#[arg(index = 1)]
-	pub entry: Option<PathBuf>,
+	#[arg(index = 1, value_delimiter = ' ', num_args = 1..=9)]
+	pub entries: Vec<PathBuf>,
```

Comment:
Why do we need `value_delimiter = ' '` here?

Reply:
yea actually it is not necessary, i think it can be deleted. the behavior is the same with or without it.